### PR TITLE
[#3539] Helm installation on minikube is incorrect

### PR DESCRIPTION
**Pull request by bilgeAiry**
Fix of the issue [#3539 ] - Updated the installation documentation for minikube and helm

### CODE REVIEW

**Review by chrismatix — DISMISSED** [review]

File: docs/docs/getting-started/installation/minikube.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The base URL for the [API](../../api/introduction.md) is the same to access the
 To customize your Minikube instance, you can adjust the `driver`, `cpus` and `memory` attributes in the following manner:
 
 ```
-airy create --provider=minikube --provider-config driver=virtualbox,cpus=4,memory=8192 my-airy
+minikube -p airy-core start --driver=docker --cpus=3 --memory=7168 --container-runtime=containerd --ports=80:80 --extra-config=apiserver.service-node-port-range=1-65535
```
Comment (ljupcovangelski) [inline docs/docs/getting-started/installation/minikube.md:54]: We don't need to change this, as the `airy` CLI also provides a way to customize the installation with the `--provider-config` flags.

**Review by ljupcovangelski — COMMENTED** [review]
Great stuff @bilgeAiry :tada: 
However the PR needs a rebase on top of the latest `develop` because there are some conflicts.

**Comment by bilgeAiry** [comment]
@ljupcovangelski review says review required after conflict resolve

**Review by ljupcovangelski — APPROVED** [review]
Nice :tada:

**Comment by bilgeAiry** [comment]
finally 👯‍♀️

**Pull request merged**